HomeWork1/main.swift

```diff
@@ -7,5 +7,111 @@
 
 import Foundation
 
-print("Hello, World!")
 
+//Создать 10 разных Dictionary с разными типами данных.Один из них должен содержать все месяца года на русском.Второй на английском
+
+var costOfFruits = ["Яблоко" : 100, "Апельсин" : 102, "Бананы" : 99, "Мандарины" : 120, "Виноград" : 150, "Манго" : 160]
```

Comment:
```suggestion
var costOfFruits = ["Яблоко": 100, "Апельсин" : 102, "Бананы" : 99, "Мандарины" : 120, "Виноград" : 150, "Манго" : 160]
```
Синтаксис в словарях, пишется как и в обьявлении типа let a: Int = 10. Ключ двоеточие пробел велью